README.rst

```diff
@@ -21,23 +21,33 @@ Quick Start Guide
 
     $ zigzag --help
 
-3. Here is an example of uploading a results JUnitXML file from Molecule::
+3. Configuring Zigzag::
+    The zigzag config file is defined by the json schema at:
+    zigzag/data/schema/zigzag-config.schema.json which is generated using a jinja2
+    template. (See zigzag/data/configs/zigzag-config-example.json for an example)
 
-    $ export QTEST_API_TOKEN="SECRET"
-    $ zigzag junit.xml 12345
+    Any test suite property from the incoming junit.xml can be interpolated, by
+    name, into the template. (See the example.)
```

Comment:
The formatting is a little off in this section. Also, you should make `zigzag/data/configs/zigzag-config-example.json` a hyperlink.